manifests/04-ca-proxy.yaml

```diff
@@ -0,0 +1,8 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  annotations:
+    config.openshift.io/inject-proxy-cabundle: "true"
```

Comment:
I noticed in https://github.com/openshift/cluster-image-registry-operator/blob/master/manifests/04-ca-config.yaml#L5-L6 that we are not marking the other CA ConfigMap as "create only" via  the annotation `release.openshift.io/create-only: "true"`. Is this right?

Reply:
It seems wrong to me, yet i don't see that CM getting stomped by the CVO.  Perhaps the CVO does not stomp configmaps back to empty?

@deads2k @abhinavdahiya ?  (tldr: we have a CM being created in the manifest for service CAs.  It's not marked create-only, but it seems to be working ok, the service-ca gets injected and the value doesn't seem to be getting stomped back out).

Reply:
(after looking at a cluster i think it is getting stomped and then re-injected.  though i'm a little surprised that isn't causing rollouts of the registry operator so we should probably confirm the registry operator is actually doing the right thing when the service ca changes.  In this case the cert doesn't actually change, but it's presumably flapping back and forth between not existing and existing, in the CM)

Reply:
I was able to dig deeper on this - there are _three_ separate control loops that are modifying the serviceca ConfigMap:

1. The CVO
2. The ca-configmap-injector
3. The image registry operator itself - https://github.com/openshift/cluster-image-registry-operator/blob/master/pkg/resource/serviceca.go#L52-L62

Will fix this in a separate PR

Reply:
good catch on (3).  seems like we should not need that generator at all?

for (1) we should just set the create-only annotation on the CM.